### DISCUSSION TITLE
install koreader on RM 2 version 3.9

### DISCUSSION THREAD

**Discussion by sife33** [toltec-dev/toltec#814] [Support] [answered]
hi everyone 
first I'M new here and I'm sorry for my English
my request is that I wanna install koreader in my remarkable 2 which have Version 3.9 
I dont konw if this is possible ; i think I can't do it without tolec after version 2.15 
note : I'm not a programmer I just know  basics things about computer that's why I hope there are some steps which I need just use ssh commend to  
install it 
thanks in advance

**Comment by Eeems** [comment, marked as answer]
https://remarkable.guide/faqs.html#can-i-downgrade-to-a-different-os-version
